src/deluge/hid/matrix/matrix_driver.h

```diff
@@ -24,6 +24,10 @@
 
 class AudioClip;
 
+struct pad_s {
+	int x, y;
+};
```

Comment:
Similar to above, this could be a class with `charToPad` as the constructor and `padToChar`/`isPad` as (static) member functions.